index.html

```diff
@@ -0,0 +1,128 @@
+        <label for="ingredient">eggs</label>
+        <br>
+        <input type="checkbox" name="ingredient" id="ingredient" value="flour">
+        <label for="ingredient">flour</label>
+        <br>
+        <input type="checkbox" name="ingredient" id="ingredient"value="bakingpowder">
+        <label for="ingredient">baking powder</label>
+        <br>
+        <input type="checkbox" name="ingredient" id="ingredient" value="vanilla">
+        <label for="ingredient">vanilla</label>
+        <br>
+        <input type="checkbox" name="ingredient" id="ingredient" value="chocolate">
+        <label for="ingredient">chocolate</label>
+        <br>
+        <input type="checkbox" name="ingredient" id="ingredient" value="heavycream">
+        <label for="ingredient">heavy cream</label>
+        <br>
+        <input type="checkbox" name="ingredient" id="ingredient" value="cocoapowder">
+        <label for="ingredient">cocoa powder</label>
+        <br>
+        <input type="checkbox" name="ingredient" id="ingredient" value="other">
+        <textarea name="ingredient" id="ingredient" cols="20" rows="1" placeholder="other"></textarea>
+      <br>
+
+ <!-- Time selector not supported by Safari browser -->
+      <h5>Preparation Time:</h5>
+      <label for="active">Active</label>
+      <input type="time" min="00:00:00" max="05:00:00"><br>
+      <label for="baking">Baking</label>
+      <input type="time" min="00:00:00" max="05:00:00"><br>
+      <label for="resting">Resting</label>
+      <input type="time" min="00:00:00" max="05:00:00"><br>
+      <label for="total">Total</label>
+      <input type="time" min="00:00:00" max="24:00:00">
+      <br>
+
+      <h5>Baking:</h5>
+      <label for="preheat">Preheat Oven</label>
+      <input type="radio" name="yes" id="preheat" value=yes>
+      <label for="yes">Yes</label>
+      <input type="radio" name="no" id="prehead" value=no>
+      <label for="no">No</label><br>
+      <label for="temperature">Temperature</label>
+      <input required type="text" size="4" name="temperature" id="temperature" placeholder="200"> °C<br>
+      <label for="total">Bake</label>
+      <input required type="time" min="00:00:00" max="05:00:00"><br>
+      
+
+      <h5>Preparation</h5>
+      <label for="instructions">Instructions:</label><br>
+      <textarea required name="instructions" id="instructions" cols="100" rows="15" placeholder="Sugar and spice and everything nice..."></textarea><br>
+
+      <label for="tipps">Extra Tipps (optional):</label><br>
+      <textarea name="extratipps" id="extratipps" cols="100" rows="5"></textarea><br><br>
+
+      <label for="imageupload">Add image (optional):</label><br>
+      <input type="file" name="image" id="imageupload"><br><br>
+
+      <label for="homepage">Recipe source (optional):</label>
+      <input type="url" id="homepage" name="homepage"><br><br>
+
+    <input type="button" value="Submit your Sweetness! &#127826" class="submit-button"><br>
```

Comment:
For this button to actually be connected to the form and submit it, you need to specify `type="submit"` instead of `type="button"`